website/docs/guides/Programmatic-API-Key-upgrade-guide-1.10.0.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: "mongodbatlas"
+page_title: "Upgrade Guide for Terraform MongoDB Atlas Provider Programmatic API Key Resource in v1.10.0"
+sidebar_current: "docs-mongodbatlas-guides-Programmatic-API-Key-upgrade-guide"
+description: |-
+MongoDB Atlas Provider : Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider: Programmatic API Key Upgrade Guide in v1.10.0
+In Terraform MongoDB Atlas Provider v1.10.0, some improvements were introduced which mainly focus on the MongoDB Atlas Programmatic API Keys (PAK) handling. This guide will help you to transition smoothly from the previous version which this resource was first released (v1.8.0) to the new version (v1.10.0).
```

Comment:
Not blocking but suggest giving a bit of info about why we are doing this. Doesn't have to be a long explanation but perhaps something like: 
```suggestion
In Terraform MongoDB Atlas Provider v1.10.0, some improvements were introduced which mainly focus on the MongoDB Atlas Programmatic API Keys (PAK) handling. The improvements introduce a much more sustainable way to manage PAK project assignment lifecycle over the long term versus being embedded in the project resource.  This guide will help you to transition smoothly from the previous version which this resource was first released (v1.8.0) to the new version (v1.10.0).
```
The reason for this is we are asking for a work here to move forward - it's much better and less likely to cause issues - but still work.

Reply:
committed but also added some more word smithing to make clear below. hope this helps customers!